src/Styles/Band.scss

```diff
@@ -0,0 +1,22 @@
+.band-container {
```

Comment:
Awesome to see that you've already changed you file extensions to `.scss` 😉 I would err on organizing your declarations idiomatically as you go so that it isn't something that has the potential to be forgotten/missed in the final days of building your app.